### PR TITLE
Tickets/dm 47204

### PR DESCRIPTION
**Pull request by teweitsai**
* Initial framework.

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -0,0 +1 @@
+Copyright 2019 Association of Universities for Research in Astronomy
```
Comment (wvreeven) [inline COPYRIGHT]: `2019` -> `2024`.

File: bin/run_rotgui [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# This file is part of ts_rotgui.
+#
+# Developed for the LSST Telescope and Site Subsystem.
```
Comment (wvreeven) [inline bin/run_rotgui:4]: Please use the modern `Vera Rubin Observatory` header here and in all other files.

**Review by wvreeven — APPROVED** [review]
LGTM. Please use the modern `Vera Rubin Observatory` header.

File: COPYRIGHT (hunk 1 above)
Comment (teweitsai) [reply COPYRIGHT]: Updated in the commit: [c90119c](https://github.com/lsst-ts/ts_rotgui/pull/1/commits/c90119c88c04df93ef3f0d07d8af7cfa121ec2ec). Thanks!

File: bin/run_rotgui (hunk 2 above)
Comment (teweitsai) [reply bin/run_rotgui:4]: Updated in the commit: [c90119c](https://github.com/lsst-ts/ts_rotgui/pull/1/commits/c90119c88c04df93ef3f0d07d8af7cfa121ec2ec). Thanks!

**Pull request merged**